dev-docs/internal-api-reference.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: page_v2
+title: Internal API Reference
+description: Internal API Reference for Prebid.js Header Bidding
+top_nav_section: dev_docs
+nav_section: reference
+pid: 10
+sidebarType: 1
+---
+
+
+
+# Internal API Reference
+
+This page has documentation for API methods that are meant for internal use by Prebid.js.
+
+{% assign api_pages = site.pages | where: "layout", "internal_api_prebidjs" %}
+
+## Find a method
```

Comment:
I'd suggest the dynamic function for finding the internal APIs from a list of 1 is probably not needed. How about removing this and re-adding when we have a screenful?

Reply:
done